docker-compose.build-override.yaml

```diff
@@ -155,6 +155,8 @@ services:
     container_name: ui
     env_file:
       - envfile
+    environment:
+      NODE_OPTIONS: "--max-old-space-size=8192"
```

Comment:
I hope this helps in making the webpack dev server not run out of heap memory when working with many (or big) files. Increased from ~4k.